prometheus_hardware_exporter/core.py

```diff
@@ -101,7 +117,7 @@ def init_default_datastore(self, payloads: List[Payload]) -> None:
                     name=payload.name, labels=payload.labels, value=0.0
                 )
 
-    def collect(self) -> Iterable[Metric]:
+    def collect(self) -> Iterable[Metric]:  # pylint: disable=R1710
```

Comment:
I do not think that disabling inconsistent return statements is right solution. You should provide `Iterable[Metric] | ...`.

Reply:
Updated